pagination/header_test.go

```diff
@@ -0,0 +1,76 @@
+package pagination
+
+import (
+	"net/http"
+	"net/url"
+	"testing"
+)
+
+func TestHeaders(t *testing.T) {
+	u, err := url.Parse("http://example.com")
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	t.Run("Create previous and first but not next or last if at the end", func(t *testing.T) {
+		h := Header(u, 120, 50, 100)
+
+		expect := http.Header{
+			"Link": []string{
+				"<http://example.com?limit=50&offset=0>; rel=\"first\"",
+				"<http://example.com?limit=50&offset=50>; rel=\"prev\"",
```

Comment:
I think the last element should not have a `,` appended

Reply:
are you referring to the comma delimiting the go slice?

Reply:
My bad 😅

Reply:
:joy: